hack/prepare-supervisor-on-kind.sh

```diff
@@ -290,6 +295,40 @@ EOF
     --dry-run=client --output yaml | kubectl apply -f -
 fi
 
+if [[ "$use_github_upstream" == "yes" ]]; then
+  # Make an GitHubIdentityProvider.  Needs to be configured with an actual GitHub App or GitHub OAuth App.
+  # TODO: claims ought be configured
+  cat <<EOF | kubectl apply --namespace "$PINNIPED_TEST_SUPERVISOR_NAMESPACE" -f -
+apiVersion: idp.supervisor.pinniped.dev/v1alpha1
+kind: GitHubIdentityProvider
+metadata:
+  name: my-github-provider
+spec:
+  claims:
+    username: id
+    groups: slug
+  client:
+    secretName: my-github-provider-client-secret
+  allowAuthentication:
+    organizations:
+      policy: AllGitHubUsers
+EOF
+
+  # Make a Secret for the above GitHubIdentityProvider to describe the GitHub client configured.
+  # TODO: make clientID and clientSecret configurable, create & store values in our secret manager
+  cat <<EOF | kubectl apply --namespace "$PINNIPED_TEST_SUPERVISOR_NAMESPACE" -f -
+apiVersion: v1
+kind: Secret
+type: "secrets.pinniped.dev/github-client"
+metadata:
+  name: my-github-provider-client-secret
+stringData:
+  clientID: REDACTED
```

Comment:
Need to get an actual github idp to pipe into this.

Reply:
Let's make these env vars like `PINNIPED_TEST_SUPERVISOR_UPSTREAM_OIDC_CLIENT_ID` so we can pass them in from other scripts or dynamically